src/nodes.coffee

```diff
@@ -1362,12 +1362,14 @@ exports.Value = class Value extends Base
   astProperties: (o) ->
     [..., property] = @properties
     property.name.csx = yes if @isCSXTag()
-    return
+    computed = property instanceof Index or property.name?.unwrap() not instanceof PropertyName
+    return {
       object: @object().ast o, LEVEL_ACCESS
-      property: property.ast o
-      computed: property instanceof Index or property.name?.unwrap() not instanceof PropertyName
+      property: property.ast o, (LEVEL_PAREN if computed)
```

Comment:
Set `LEVEL_PAREN` when this is a "computed" access eg `a[b]` - this corresponds to the `LEVEL_PAREN` used in `Index::compileNode()`